scipy/stats/_qmc.py

```diff
@@ -64,7 +78,11 @@ def check_random_state(seed=None):
                          ' instance' % seed)
 
 
-def scale(sample, l_bounds, u_bounds, reverse=False):
+def scale(
+        sample: np.typing.ArrayLike,
+        l_bounds: np.typing.ArrayLike,
+        u_bounds: np.typing.ArrayLike,
+        reverse: bool = False) -> np.ndarray:
```

Comment:
Careful about the extra spaces and the ending comma. It should be like this
```suggestion
def scale(
    sample: np.typing.ArrayLike,
    l_bounds: np.typing.ArrayLike,
    u_bounds: np.typing.ArrayLike,
    reverse: bool = False
) -> np.ndarray:
```

Reply:
Can we at least do `from numpy.typing import ArrayLike`. That's already lots of repetition.

Reply:
True. He removed it because of the import error. But I guess it was a better choice in the first place to use the following (as it was in the pyi)
```python
if TYPE_CHECKING:
    import numpy.typing as npt
```